build.sbt

```diff
@@ -266,7 +280,7 @@ lazy val postgres = project
       import doobie._, doobie.implicits._
       import doobie.postgres._, doobie.postgres.implicits._
       implicit val cs = IO.contextShift(scala.concurrent.ExecutionContext.global)
-      val xa = Transactor.fromDriverManager[IO]("org.postgresql.Driver", "jdbc:postgresql:world", "postgres", "")
+      val xa = Transactor.fromDriverManager[IO]("org.postgresql.Driver", "jdbc:postgresql:world", "postgres", "password")
```

Comment:
Not opposed to this change but can you split this out to a separate PR for me? :slightly_smiling_face:

Reply:
https://github.com/tpolecat/doobie/pull/1792